test/slicing/test_convergence.py

```diff
@@ -14,31 +16,42 @@
 from snorkel.slicing.utils import add_slice_labels, convert_to_slice_tasks
 from snorkel.types import DataPoint
 
-SEED = 123
-
 
-# TODO: update me!
+# Define SFs specifying points inside a circle
 @slicing_function()
 def f(x: DataPoint) -> int:
-    return 1 if x.x1 > x.x2 + 0.5 else 0
+    radius = 0.3
+    h, k = (-0.15, -0.3)  # center
+    return np.sqrt((x.x1 - h) ** 2 + (x.x2 - k) ** 2) < radius
 
 
 @slicing_function()
 def g(x: DataPoint) -> int:
-    return 1 if x.x1 > x.x2 + 0.3 else 0
+    radius = 0.3
+    h, k = (0.25, 0.0)  # center
+    return np.sqrt((x.x1 - h) ** 2 + (x.x2 - k) ** 2) < radius
 
 
-N_TRAIN = 1000
-N_VALID = 100
+SEED = 123
```

Comment:
This isn't reused, just put in set_seed call

Reply:
see update, now it's used in both tests